config/env/test.json

```diff
@@ -58,6 +58,7 @@
     "awsRegion": "us-east-1",
     "sqsQueueUrl": "",
     "pollingIntervalMS": 10000,
+    "maxAttemptsToRetrieveMessages": 0,
```

Comment:
This will cause workers to wait forever? That's ok to do for tests?

Reply:
The tests should timeout before then. Prior to this, the attempt were unlimited.